cmd/sebak/cmd/wallet/payment.go

```diff
@@ -103,24 +103,29 @@ func init() {
 			// Check that account's balance is enough before sending the transaction
 			{
 				newBalance, err = senderAccount.GetBalance().Sub(amount)
-				if err == nil {
-					newBalance, err = newBalance.Sub(common.BaseFee)
-				}
-
 				if err != nil {
-					fmt.Printf("Attempting to draft %v GON (+ %v fees), but sender account only have %v GON\n",
-						amount, common.BaseFee, senderAccount.GetBalance())
+					fmt.Printf("Attempting to draft %v GON, but sender account only have %v GON\n",
+						amount, senderAccount.GetBalance())
 					os.Exit(1)
+				} else {
+					if !(flagFreeze || (senderAccount.Linked != "")) {
```

Comment:
I figure, this is better to read :)
`if !flagFreeze && (senderAccount.Linked == "")`